src/dict/stardict.cc

```diff
@@ -436,14 +436,15 @@ string StardictDictionary::handleResource( char type, char const * resource, siz
     {
       QString articleText = QString( "<div class=\"sdct_h\">" ) + QString::fromUtf8( resource, size ) + "</div>";
 
-      static QRegularExpression imgRe( R"((<\s*(?:img|script)\s+[^>]*src\s*=\s*["']?)(?!(?:data|https?|ftp):))",
-                                       QRegularExpression::CaseInsensitiveOption );
-      static QRegularExpression linkRe( R"((<\s*link\s+[^>]*href\s*=\s*["']?)(?!(?:data|https?|ftp):))",
-                                        QRegularExpression::CaseInsensitiveOption );
-
-      articleText.replace( imgRe, "\\1bres://" + QString::fromStdString( getId() ) + "/" )
-        .replace( linkRe, "\\1bres://" + QString::fromStdString( getId() ) + "/" );
-
+      static QRegularExpression imgRe(
```

Comment:
Let's document this thing for future.

```suggestion
      // Replace urls that is relative path but not direct data or https
      // Match: <link href=abc.png/>
      // Match:  <img src='abc.png'/>
      // Not Match: <link href='http://abc.png'/>
      // Not Match: <link href='data:image/jpeg;.......'/>
      static QRegularExpression imgRe(
```